src/main/java/seedu/address/model/event/StartTime.java

```diff
@@ -28,28 +30,28 @@ public class StartTime implements Comparable<StartTime> {
     public StartTime(String startTime) {
         requireNonNull(startTime);
         assert !startTime.isBlank();
-        checkArgument(isValidStartTime(startTime), MESSAGE_FORMAT_CONSTRAINTS);
+        checkArgument(isValidStartTimeFormat(startTime), MESSAGE_FORMAT_CONSTRAINTS);
         checkArgument(isValidStartTimeValue(startTime), String.format(MESSAGE_VALUE_CONSTRAINTS, startTime));
-        String [] parsedTime = startTime.split(":", 2);
+        String [] parsedTime = startTime.split(TIME_DELIMITER, 2);
         this.time = LocalTime.of(Integer.parseInt(parsedTime[0]), Integer.parseInt(parsedTime[1]));
     }
 
     /**
      * Checks if a given string follows the valid StartTime input format.
      * @return boolean value.
      */
-    public static boolean isValidStartTime(String test) {
+    public static boolean isValidStartTimeFormat(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
     /**
      * Checks if a given string can be parsed to a valid StartTime value.
-     * @param test string input, it is guaranteed that the format check for test is done before this function.
+     * @param test string input, the format check for test should be done before calling this function.
      * @return boolean value.
      */
     public static boolean isValidStartTimeValue(String test) {
         try {
-            String [] parsedTime = test.split(":", 2);
+            String [] parsedTime = test.split(TIME_DELIMITER, 2);
```

Comment:
Extra space after `String`

```suggestion
            String[] parsedTime = test.split(TIME_DELIMITER, 2);
```

Reply:
Thanks for spotting this! i forgot to change this line